### ISSUE TITLE
Building the Install Package empties the "changed-files" file

### ISSUE THREAD

**Issue by eSilverStrike** [Geeklog-Core/geeklog#978]
After building the Geeklog Install package for v2.2.1 Beta 1 I notice that the "changed-files" file in the docs directory is empty.

@mystralkk  The "changed-files" file seems to update fine I notice when you had a commit on Sept, 28, 2019. 

https://github.com/Geeklog-Core/geeklog/commit/f90c7fad42ad95b4eed577f788389085e0ae4b75

I thought this file would update when I run the following command:

phing.bat dist 

or must I do something else first?

I am wondering if it should be like the language synch where I synch the "changed-files" before the release so it can be recorded in the repository as such (before I tag the commit as release in GitHub)… I guess I can always commit the changed-files file  after creating the install package and then tag the commit in GitHub to mark the release.

**Comment by mystralkk** [comment]
The "changed-files" file is created by using git commands, so you have to install git commands on your PC.

**Comment by eSilverStrike** [comment]
Right I forgot about installing Git on my new development server. Okay I installed git and the changed-files gets updated and the process completes but I get the following warning via the command prompt.

> warning: inexact rename detection was skipped due to too many files.
> warning: you may want to set your diff.renameLimit variable to at least 1692 and retry the command.

Now the changed-files updated and it looks to be correct but I am not sure it is missing files because of the limit so I added a fix to up the renamelimit.
